src/templates/static/special/rebootLocalHost.yml

```diff
@@ -0,0 +1,17 @@
+# Reboots a system if any of the preceeding plays sets the 'insights_needs_reboot' variable to true.
+# The variable can be overridden to suppress this behavior.
+- name: Reboot localhost
+  hosts: "@@HOSTS@@"
+  vars:
+    insights_signature_exclude: "/hosts"
+  become: True
+  gather_facts: False
+  tasks:
+    - when:
+        - insights_needs_reboot is defined
+        - insights_needs_reboot
+      block:
+        - name: Reboot system
+          reboot:
```

Comment:
I don't think this is going to work for localhost as the machine running the playbook is the one that gets rebooted. Have you tested this?

Reply:
@jharting I have, and I now see that this module does not run for the reason you illustrate above.  Looks like I'll change it back to:
> sleep 2 && shutdown -r +1

Reply:
I've replaced the logic with the one I stated above.  Additionally based on the conversation with Link I added the "insights-client post reboot" step to create the file designated in this unit file PR:
https://github.com/RedHatInsights/insights-client/pull/138/files